packages/addon-dev/src/rollup-keep-assets.ts

```diff
@@ -14,12 +14,6 @@ export default function keepAssets({
   return {
     name: 'copy-assets',
 
-    // Prior to https://github.com/rollup/rollup/pull/5270, we cannot call this
```

Comment:
From what I can see this issue got resolved

Reply:
They have a nifty bot that tells you what version of rollup includes the PR 🎉 apparently this was fixed in `rollup@4.6.0`.

I wonder if we have any peer dependency reference or anything that would allow us to say which versions of rollup are supported 🤔

Reply:
also if we are changing the "rollup support matrix" this one change could technically make this a breaking change... maybe 🤔

Reply:
@mansona that is a good point, I can add `rollup` as a peerDep to `@embroider/addon-dev`'s package.json file if you feel like that is the right approach?
And it might we a major release, if we take that into account?

Reply:
so the fact that rollup does it really makes me feel like we should do the same thing. And we need to declare the minimum version of rollup that we support

if you want to go ahead and add that in this PR we can mark it as breaking and it will automatically get picked up and released correctly 👍

Reply:
@mansona I added the change, take a look when you can